includes/WeMail.php

```diff
@@ -411,7 +411,7 @@ private function set_wemail_api() {
          *
          * @param string
          */
-        $api = apply_filters( 'wemail_api_url', 'https://api.getwemail.io/v1' );
+        $api = apply_filters( 'wemail_api_url', 'http://wemail-api.test/v1' );
```

Comment:
You just broke the whole weMail by changing this value. If you need to change `wemail_api_url` on the development you need to just use `weMail Dev Tools` plugin.